functions.php

```diff
@@ -0,0 +1,43 @@
+<link rel="stylesheet" href="collector.css">
+
+<?php
+
+function dbpull() {$connectionString = 'mysql:host=db; dbname=winecollector';
+    $dbUsername = 'root';
+    $dbPassword = 'password';
+    $db = new PDO($connectionString, $dbUsername, $dbPassword);
+
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+    $queryString = 'SELECT * FROM `wines` ORDER BY `date` DESC';
```

Comment:
Replace * with the names of the columns (don't forget backticks!).

It is not recommended to have * in production code.